package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "hmrc-frontend",
-  "version": "6.78.0",
+  "version": "6.79.0",
   "description": "Design patterns for HMRC frontends",
   "scripts": {
     "start": "gulp dev",
+    "test:compatibility": " jest tasks/gulp/__tests__/check-compatibility-with-govuk-frontend-version.test.js",
```

Comment:
should we link this into another test task, so that we don't have to remember to run it if we are doing `npm test`